rust/trace/src/lib.rs

```diff
@@ -240,12 +240,78 @@ impl Config {
 static SAMPLE_COUNTER: AtomicUsize = ATOMIC_USIZE_INIT;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
-pub enum SampleType {
-    /// This is an instantaneous sample (i.e. X occurred)
+pub enum SampleEventType {
```

Comment:
I'm assuming this is 1:1 with the chrome trace spec?

Reply:
I tried to make it so, so if it doesn't it's a bug.